packages/artifact/src/internal/utils.ts

```diff
@@ -74,7 +74,8 @@ export function isRetryableStatusCode(statusCode?: number): boolean {
     HttpCodes.BadGateway,
     HttpCodes.ServiceUnavailable,
     HttpCodes.GatewayTimeout,
-    HttpCodes.TooManyRequests
+    HttpCodes.TooManyRequests,
+    413 // Payload Too Large
```

Comment:
Hardcoded for now. Will update in the future once a new version of `actions/http-client` is out: https://github.com/actions/http-client/pull/31